scripts/sync_pop_candidate.sh

```diff
@@ -77,7 +77,11 @@ function pop() {
     make manifests
     git add "${subtree_dir}" "${ROOT_GENERATED_PATHS[@]}"
     git status
-    git commit --amend --allow-empty --no-edit --trailer "Upstream-repository: ${remote}" --trailer "Upstream-commit: ${rc}"
+
+    upstream_repository=$'\n\nUpstream-repository:'
+    upstream_commit=$'\nUpstream-commit:'
+    git commit --allow-empty --amend -m "$(git log --format=%B -n 1) $upstream_repository $remote"
+    git commit --allow-empty --amend -m "$(git log --format=%B -n 1) $upstream_commit $rc"
```

Comment:
We already talked in slack, but we have control over the container image that gets used here, so could we avoid hacking around the --trailer flag only being present in more recent git versions by curling the git version we need and building from source?

Reply:
That sounds like a good plan. I updated the other PR to actually do the work of `--trailer` properly but without using the flag. It would be a good follow-up to bring it back and update the image though.